### DISCUSSION TITLE
Problema de Certidão Positiva Municipal

### DISCUSSION THREAD

**Discussion by anadulce** [apyb/comunidade#175] [Pergunta]
Boa tarde, pessoal! Tudo bem?

Enquanto estávamos emitindo as documentações para inscrições via empenho, chegou essa informação:

> A certidão municipal da empresa apontou situação de "Certidão Positiva". Desta forma, solicitamos que seja regularizada a situação e enviada certidão negativa para que possamos prosseguir com a instrução processual da contratação a que diz respeito a minuta de ETP para Contratação da Associação Python Brasil, CNPJ n.º 09.344.485/0001-05, para ministrar a ação de capacitação PythonCerrado + Plone Conference Brasília.

Poderiam conferir com a contabilidade se a APyB está com alguma pendência? E de onde parte essa certidão positiva? Se for necessário, posso pedir mais informações.

@apyb/diretoria

**Comment by anadulce** [comment]
Existe outro órgão nos cobrando a certidão negativa da prefeitura onde a Associação Python Brasil está inscrita, eu imagino que seja pelo mesmo motivo do e-mail acima.

Assim que for resolvido, poderiam emitir outra certidão negativa, por favor?
Eu enviei uma solicitada recentemente, mas aparentemente, ela está desatualizada.

@apyb/diretoria

**Comment by anicelysantos** [comment]
@anadulce vamos verificar com a contadora, mas como amanhã é feriado, só conseguiremos vê esta questão a partir da quinta (21/11).

**Reply by anadulce** [reply]
Muito obrigada, Any <3

**Comment by anicelysantos** [comment]
@anadulce aqui está: https://drive.google.com/file/d/1Giz4Fd3eM5Q12mxNfvmnF9BZLrmvkt39/view?usp=drive_link

**Reply by anadulce** [reply]
Muuuito obrigada, Any!!
